.travis.yml

```diff
@@ -1,5 +1,7 @@
+sudo: false
 language: node_js
 node_js:
- - '0.10'
```

Comment:
For reference, here's where it fails on 0.10:

> https://travis-ci.org/kss-node/kss-node/jobs/91464531#L505-L507
```
kss.parse() given options .typos: Misspelt experimental should find correct spelling:
     TypeError: Cannot call method 'section' of undefined
      at Context.<anonymous> (test/test_parse.js:541:28)
```

Thanks, updated in https://github.com/kss-node/kss-node/commit/d129f11fb600fcd63f503b599fab94d24cc1e875 .